build-system-tests/templates/components/react/next/template-tsconfig-11.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Was this file autogenerated?

Reply:
It was modified from the auto-generated nextjs 13